Makefile

```diff
@@ -6,7 +6,10 @@ BUILD_DATE := $(shell date -u +"%Y-%m-%d")
 BUILD_TIMESTAMP := $(shell date -u +"%Y-%m-%dT%H:%M:%S%Z")
 VCS_BRANCH := $(strip $(shell git rev-parse --abbrev-ref HEAD))
 VCS_REF := $(strip $(shell [ -d .git ] && git rev-parse --short HEAD))
-DOCKER_REPO ?= quay.io/observatorium/observatorium-operator
+CONTAINER_REPO ?= quay.io/observatorium/observatorium-operator
+CONTAINER_ENGINE ?= docker
```

Comment:
@dofinn you could change this too `CONTAINER_ENGINE=$(shell command -v podman 2>/dev/null || command -v docker 2>/dev/null)` to auto select the container engine like we do for our builds. Non blocking comment, just passing by  :slightly_smiling_face:

Reply:
Thanks. Updated PR to include this. Much nicer then exporting CONTAINER_ENGINE via ~/.bash\|zshrc